src/models/ontotext-yasgui/query-executed-event.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Model for event of type {@link EventDataType.QUERY_EXECUTED} emitted by "ontotext-yasgui-web-component".
+ */
+export class QueryExecutedEvent {
+    /**
+     * Constructs the model for {@link EventDataType.QUERY_EXECUTED} event.
+     *
+     * @param {EventData} eventData - event emitted by "ontotext-yasgui-web-component".
+     */
+    constructor(eventData) {
+        this.TYPE = eventData.TYPE;
```

Comment:
Is this the query type? It's not quite obvious what it is

Reply:
This is the type of the event. All events we implemented has TYPE field that distinguish them each other.